MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
+   - Merges in the PR.
 
-### MIP0c3: The MIP Lifecycle
+3. **Request for Comments (RFC)**: A period of reviewing by the community and attendant redrafting begins. The minimum duration of this period is determined by two variables:
 
-**The MIP Lifecycle Flow and MIP Statuses**
+   - `Feedback Period`: 3 months
+   - `Frozen Period`: 1 month
 
-![mip_life_cycle](https://user-images.githubusercontent.com/32653033/79086728-19d93900-7d0b-11ea-8086-c255d919096c.png)
+   These periods can overlap. See `MIP0c1` for their definitions
 
+4. **Fulfilled Feedback Period Requirements:** After the MIP has fulfilled the RFC phase, it is ready for Formal Submission.
 
-**MIP Status Criteria**  
+5. **Formal Submission (FS):** At this point the MIP Author submits their MIP to the Governance Cycle by posting it to the [*Formal Submission* forum category](https://forum.makerdao.com/c/mips/fs/16) within the [formal submission window of a Governance Cycle](https://github.com/makerdao/mips/blob/master/MIP16/mip16.md).
 
-**1. Conception:** The lifecycle of a MIP begins when the MIP proposal is posted on the Maker forum. However, in order for a MIP to move to the next stage, it needs to satisfy the transition criteria (1) described below:
+   > A MIP can be re-submitted to the formal submission process a maximum of 2 additional times (3 total), without having to go through phase 1- 4 again, if it failed to pass due to legitimate external reasons (e.g. got bundled in a governance poll or executive vote with a controversial proposal - subject to the governance facilitators judgement).
 
--   Submitted to the MIPs Discourse Forum.
--   Submitted to the MIPs Github repository (with a Pull Request created by the MIP Author or MIP Editor).
--   The format must follow the appropriate MIP Template for its type.
--   MIPs must be original or replacement versions of older MIPs (No repeats allowed).
+6. **Approved by the Governance Facilitator(s):** The Governance Facilitators evaluate the MIP. If they approve it, the MIP will be entered in the inclusion poll of the Governance Cycle. If they do not, the MIP may be reconsidered to enter the Governance Cycle at a later date.
 
-**2. Approved by MIP Editor(s):** This phase of a MIP’s life cycle is when the MIP Editor(s) confirms that the proposed MIP follows the correct structure and editorial criteria defined in the MIP template. If the criteria is not met, the MIP Editor(s) will provide an explanation to the MIP Author as to why and allow them to make the appropriate changes before reconsideration. If the criteria have been met, the MIP Editor(s) performs the following actions:
-    
--   The MIP is approved by a MIP Editor(s) and is assigned a formal MIP number.
--   The PR is merged in by a MIP Editor(s).
+7. **Governance Cycle:** At this point, MKR holders vote on whether to include the MIP in the governance poll, ultimately determining whether or not the MIP can formally enter the Governance Cycle.
```

Comment:
Refer to MIP3 here. That explains the Governance Cycle.

Reply:
Actually this is just misleading. It's in the Governance Cycle at this point already.

Reply:
It should be something more along the lines of 'At this point it enters the Monthly Governance Cycle and proceeds  through a number of votes until it is either accepted or rejected. Further details in MIP3 (or something.)'